sdk/subscription/azure-mgmt-subscription/setup.py

```diff
@@ -1,87 +1,37 @@
-#!/usr/bin/env python
-
-#-------------------------------------------------------------------------
+# coding=utf-8
+# --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#--------------------------------------------------------------------------
-
-import re
-import os.path
-from io import open
-from setuptools import find_packages, setup
-
-# Change the PACKAGE_NAME only to change folder and different name
-PACKAGE_NAME = "azure-mgmt-subscription"
-PACKAGE_PPRINT_NAME = "Subscription Management"
-
-# a-b-c => a/b/c
-package_folder_path = PACKAGE_NAME.replace('-', '/')
-# a-b-c => a.b.c
-namespace_name = PACKAGE_NAME.replace('-', '.')
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+# coding: utf-8
 
-# azure v0.x is not compatible with this package
-# azure v0.x used to have a __version__ attribute (newer versions don't)
-try:
-    import azure
-    try:
-        ver = azure.__version__
-        raise Exception(
-            'This package is incompatible with azure=={}. '.format(ver) +
-            'Uninstall it with "pip uninstall azure".'
-        )
-    except AttributeError:
-        pass
-except ImportError:
-    pass
+from setuptools import setup, find_packages
 
-# Version extraction inspired from 'requests'
-with open(os.path.join(package_folder_path, '_version.py'), 'r') as fd:
-    version = re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]',
-                        fd.read(), re.MULTILINE).group(1)
+NAME = "azure-mgmt-subscription"
+VERSION = "1.0.0b1"
 
-if not version:
-    raise RuntimeError('Cannot find version information')
+# To install the library, run the following
+#
+# python setup.py install
+#
+# prerequisite: setuptools
+# http://pypi.python.org/pypi/setuptools
 
-with open('README.md', encoding='utf-8') as f:
-    readme = f.read()
```

Comment:
We shoudl restore the majority of this `setup.py` to what it was before. A lot of the metadata has been lost.